app/models/user.rb

```diff
@@ -3,7 +3,7 @@ class User < ApplicationRecord
   has_many :posts, foreign_key: 'author_id'
   has_many :likes, foreign_key: 'user_id'
 
-  def show_post_user(user)
-    Comment.where(user_id: user.id).last(3)
+  def user_recent_posts(user)
+    Post.where(author_id: user.id).order('created_at Desc').limit(3)
   end
```

Comment:
- [ ] Great job implementing the changes suggested by the previous reviewer. However, The `user_recent_posts` method still needs some adjustments. This method is to return the 3 most recent posts created in the database. I can see you're returning the 3 most recent posts but you're returning the post based on the author_id which shouldn't be so. 
    - If you look at the previous reviewer's comment your would see he provided a clear example of how to achieve this.